README.md

```diff
@@ -0,0 +1,183 @@
+<a href="https://sktime.org"><img src="https://github.com/alan-turing-institute/sktime/blob/main/docs/source/images/sktime-logo-no-text.jpg?raw=true)" width="175" align="right" /></a>
+
+# Welcome to sktime
+
+> A unified interface for machine learning with time series
+
+:rocket: **Version 0.7.0 out now!** [Check out the release notes here.](https://github.com/alan-turing-institute/sktime/releases)
+
+sktime is a library for time series analyis in Python. It provides a unified interface for multiple time series learning tasks, including time series classification, regression, clustering, annotation and forecasting. It comes with time series algorithms and [scikit-learn] compatible tools to build, tune and validate time series models.
```

Comment:
Typo in "for time series analyis" - `analySis`.
Are there other ML tasks that can be solved with our tools, except the ones counted here? If there aren't then we could just say "It provides a unified interface for time series classification, regression, clustering, annotation and forecasting."

Reply:
Not yet, but hopefully in the future, I'll update it to say that this is what we currently support - thanks!